libcipher/caesar.py

```diff
@@ -0,0 +1,76 @@
+__author__ = 'kskrishnasangeeth'
+
+import string
+
+
+def __cipher_helper(message, key):
+    """Function where the encryption/decryption logic gets executed.
+
+    Args:
+        message (str): Message that is supplied for encryption/decryption.
+        key (int): Key used for encryption/decryption.
+
+    Returns:
+        message (str): Ciphered Message in lowercase.
+
+    """
+
+    cipher_text = ""
+    for each_char in message.lower():
+        char_pos = string.ascii_lowercase.find(each_char)
+        if char_pos == -1:
+            cipher_text += each_char
+            continue
+        else:
+            char_pos = (char_pos + key) % 26
+            cipher_text += string.ascii_lowercase[char_pos]
+    return cipher_text
+
+
+def encrypt(message, key):
+    """Returns the encrypted message for the string passed as argument.
+
+    Uses a random key for generating the cipher text
+
+    Args:
+        message (str): The message to be encrypted.
+        key (int): The key based on which encryption is performed.
+
+    Returns:
+        str: the encrypted message.
+
+    """
+    __check_exceptions(key)
+    encrypt_text = __cipher_helper(message, key)
+    return encrypt_text
+
+
+def decrypt(message, key):
+    """Returns the actual message which was encrypted
+       with caesar cipher using Key.
```

Comment:
Make this is a single line. If you want a more detailed description that should be provided in a separate paragraph below the one-line description as shown below.

    def myfunc(arg1, arg2):
        """"My one-line description.

        Multi-line description is in a separate paragraph, separated from
        the one-line description by a blank line.

        Args:
               ....
        """"